application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -292,16 +398,23 @@ private boolean hasTagManageRole(@NotNull Member member) {
 
     private enum Subcommand {
         RAW("raw"),
-        CREATE("create"),
-        CREATE_WITH_MESSAGE("create-with-message"),
-        EDIT("edit"),
-        EDIT_WITH_MESSAGE("edit-with-message"),
-        DELETE("delete");
+        CREATE("create", "created"),
+        CREATE_WITH_MESSAGE("create-with-message", "created"),
+        EDIT("edit", "edited"),
+        EDIT_WITH_MESSAGE("edit-with-message", "edited"),
+        DELETE("delete", "deleted");
 
         private final String name;
+        private final String actionVerb;
 
         Subcommand(String name) {
             this.name = name;
+            this.actionVerb = "";
+        }
```

Comment:
* Use constructor forwarding instead `Subcommand(String name) { this(name, ""); }`
* Why even support that case? Lets remove this constructor all together

Reply:
its for the `RAW` subcommand, that doesn't need a verb and doesn't get logged.

Reply:
Yeah but it still gets `""` and the scope of this class is so minimal that you might as well just do `RAW("raw", "")` and then get rid of this constructor.

Reply:
done.